packages/fluentui/react-northstar/test/specs/components/Tree/Tree-test.tsx

```diff
@@ -71,7 +71,7 @@ const checkOpenTitles = (wrapper: ReactWrapper, expected: string[]): void => {
 };
 
 describe('Tree', () => {
-  isConformant(Tree, { autoControlledProps: ['activeItemIds'] });
+  isConformant(Tree, { constructorName: 'Tree', autoControlledProps: ['activeItemIds'] });
```

Comment:
```suggestion
  isConformant(Tree, { constructorName: 'Tree', autoControlledProps: ['activeItemIds', 'selectedItemIds'] });
```
Let's test all autocontrolled props